tests/test_scripts.py

```diff
@@ -1926,3 +1926,15 @@ def test_readstats_empty():
     assert status == 0
 
     assert expected_output in out
+
+
+def test_existance_failure():
+    expected_output = 'ERROR: Input file'
+
+    args = ['thisfiledoesnotexistatall']
+
+    status, out, err = utils.runscript(
```

Comment:
Will the test in test_scripts.py work to as the first test you described?  If I'm understanding you, there only needs to be another test in test_script_arguments.py to test the code in kfile.  I would like some expansion on the test_script_arguments.py test if possible.

Also - how do I a run a spell checker? Thanks!